src/bloom.cpp

```diff
@@ -366,7 +368,9 @@ bool CRollingBloomFilter::contains(const uint256 &hash) const
 bool CRollingBloomFilter::contains(const COutPoint &outpoint) const { return contains(ToVector(outpoint)); }
 void CRollingBloomFilter::reset()
 {
+#ifndef ANDROID // On Android don't pick a new tweak value because we don't have GetRand
     nTweak = GetRand(std::numeric_limits<unsigned int>::max());
```

Comment:
is this because OpenSSL `RAND_bytes` is not available? Isn't there any other method to get a random value while building for Andorid? 
this discussion seems relevant: https://github.com/arvidn/libtorrent/issues/3103

Reply:
Note: remove the entire rolling bloom filter from java/android since its not being used.

Reply:
done